devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java

```diff
@@ -46,6 +46,7 @@ public QuarkusPluginExtension(Project project) {
     }
 
     public void beforeTest(Test task) {
+        project.getLogger().info("QuarkusPluginExtension.beforeTask " + task.getName());
```

Comment:
The issue is that loading an app model this way is known to be slow. It could take 5 sec, for example, or even longer. I added it just for that reason and because it's only for launching from an IDE use-case.

Reply:
Well, in that case, it should be a message that an end user can understand such as `Loading the application model...` or something similar?

Reply:
Oh, sorry! I thought of a different info message I added in another place. Yes, this is a leftover, thanks for spotting it!